source/linux/basic/usersystem.rst

```diff
@@ -0,0 +1,70 @@
+Usersytem Basics
+================
+
+Linux has a multiuser user system which allows multiple users to work on one system at the same time. 
+Linux' user system includes a permission system (more on that on another Page). The following commands will give you a basic overview:
+
+adduser <username>
+^^^^^^^^^^^^^^^^^^
+    Description
+        Adds a user with the given username	
+    
+    Example Usage
+
+        .. code-block:: bash
+
+            host$ adduser demouser
+
+passwd <username>
+^^^^^^^^^^^^^^^^^
+    Full Name
+        **Passw** or **d**
+        
+    Description
+        Changes a users password	
+
+    Example Usage
+        .. code-block:: bash
+
+            host$ passwd demouser
+
+sudo <command>
+^^^^^^^^^^^^^^^^^
+    Full Name
+        **Su** peruser **Do**
+
+    Description
+        Executes a Command with superuser privileges.
+        **Warning:** only execute a command with superuser privileges, if you know exactly what you are doing!
+
+        Sometimes you will notice, that you cannot execute a command, because you does not have all permissions. 
+        If you execute it with sudo, you should have all needed permissions. (e.g. passwd has to be run with sudo)
+	
+    Example Usage
+        .. code-block:: bash
+
+            host$ sudo passwd demouser
```

Comment:
Maybe we should use a other example, like apt, since passwd can also be used without super user privileges, as long as it's your own user.